datastore/mock_test.go

```diff
@@ -120,8 +120,8 @@ func (s *mockServer) popRPC(gotReq proto.Message) (interface{}, error) {
 
 		if !proto.Equal(gotReq, ri.wantReq) {
 			return nil, fmt.Errorf("mockServer: bad request\ngot:\n%T\n%s\nwant:\n%T\n%s",
-				gotReq, proto.MarshalTextString(gotReq),
-				ri.wantReq, proto.MarshalTextString(ri.wantReq))
+				gotReq, gotReq,
+				ri.wantReq, ri.wantReq)
```

Comment:
I'm not sure that the protos will render via `%s` all that well, at least not in the textproto format as was done before...is it worth just adding the https://pkg.go.dev/google.golang.org/protobuf@v1.33.0/encoding/prototext usage?

Not a blocker though...

Reply:
will do .